pages/price-feeds/getting-started.mdx

```diff
@@ -0,0 +1,15 @@
+## Getting Started
+
+Integrating Pyth price feeds into your application is quick and easy.
+
+### Build 🧑‍💻
```

Comment:
No emojis please -- our brand is serious / institutional / high-trust, not cutesy.